style.css

```diff
@@ -663,6 +850,96 @@
     margin-left: -32px;
   }
 
+  .social-pices {
+    display: flex;
+    gap: 15px;
+    padding-left: 0;
+    margin-top: 25px;
+  }
+
+  .social-pices li {
+    list-style-type: none;
+  }
+
+  .project-technologies li {
+    margin-bottom: 5px;
+    padding-top: 2px;
+    font-family: "Poppins", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    list-style-type: none;
+    text-align: center;
+    color: #6070ff;
+    width: 49px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer kindly consider making sure that you style of your technologies section matches that of the `Figma` design as it currently doesn't.  To style it to match the `Figma` design kindly consider the below suggestion:
  - [ ] Remove the styles `padding-top`, `line-height`, `letter-spacing`, `width`, and `height` and add a `padding: 5px 12px`
  - [ ] Also, consider removing the pseudo-classes you have on lines `881` and `905` as they are distrusting your technologies from aligning to the center.
  
  See below for more details:
  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="288" alt="Screenshot 2023-06-23 at 15 13 09" src="https://github.com/Danny-imo/my-project/assets/107192076/ceb52b45-69fb-4d5c-833f-bce3ecefc241">
</td>
<td>
<img width="288" alt="Screenshot 2023-06-23 at 15 13 26" src="https://github.com/Danny-imo/my-project/assets/107192076/821d788e-29f0-42e8-9d9f-a00d82211161">
</td>
</tr>
</table>
  
  By implementing the suggested changes above your design should look much better and close to the `Figma` design template as much as possible.